stage1/x86_fsp.mk

```diff
@@ -1,3 +1,9 @@
+SIGN_TOOL?=../tools/keytools/sign
+SIGN_OPTIONS?=--ecc256 --sha256
+SIGN_KEY?=../wolfboot_signing_private_key.der
+X86FSP_PATH?=../src/x86
```

Comment:
Please elaborate. What other cases should we support? Aren't the path of the fsp hardcoded in the generation script?

Reply:
the downloading script is hardcoded indeed, but the config has the full path and it can be changed. 
I suggest either supporting arbitrary paths here or removing the full path from the config and using just the name.